pkg/scheduler/admission/podnodeconstraints/admission.go

```diff
@@ -77,6 +78,9 @@ var resourcesToCheck = map[unversioned.GroupResource]unversioned.GroupKind{
 // we choose not to handle in this plugin
 var resourcesToIgnore = []unversioned.GroupKind{
 	extensions.Kind("DaemonSet"),
+	securityapi.Kind("PodSpecSelfSubjectReview"),
```

Comment:
Drop a TODO in here? We may actually want to default these at some later date, so maybe even an issue.